bin/i18n/resources/dashboard/marketing_announcements.rb

```diff
@@ -0,0 +1,22 @@
+Dir[File.expand_path('../marketing_announcements/**/*.rb', __FILE__)].sort.each {|file| require file}
```

Comment:
```suggestion
require_relative '../../i18n_script_utils'
require_relative '../dashboard'

Dir[File.expand_path('../marketing_announcements/**/*.rb', __FILE__)].sort.each {|file| require file}
```

Reply:
Question on this, does a child class inherit the variables/methods of the parent?

Reply:
Yes, they do, but we should still require them there for clarity

Reply:
Yes, they do, but we should require them there for clarity